src/spanningtrees/planar_maximally_filtered_graph.jl

```diff
@@ -0,0 +1,74 @@
+"""
+	planar_maximally_filtered_graph(g)
+
+Compete the Planar Maximally Filtered Graph (PMFG) of weighted graph `g`.
+
+### Examples
+```
+using Graphs, SimpleWeightedGraphs
+N = 20
+M = Symmetric(randn(N, N))
+g = SimpleWeightedGraph(M) 
+p_g = planar_maximally_filtered_graph(g)
+```
+
+### References 
+- Tuminello et al. 2005
+"""
+function planar_maximally_filtered_graph end
+@traitfn function planar_maximally_filtered_graph(
+    g::AG::(!IsDirected), distmx::AbstractMatrix{T}=weights(g); minimize=true
+) where {T<:Real,U,AG<:AbstractGraph{U}}
+    #check graph is not directed
+    if is_directed(g)
+        error("PMFG only supports non-directed graphs")
```

Comment:
This check is not necessary, the trait `g::AG::(!IsDirected)` from above already prevents this function from being called with directed graphs.